.github/PULL_REQUEST_TEMPLATE/bug_PR.md

```diff
@@ -0,0 +1,101 @@
+---
+name: Bug PR
+about: Submit a PR for a bug
+title: 'BUG: '
+labels: bug
+assignees: ''
+
+---
+### About this repo
+- `q2-moshpit` is a `qiime2` plugin that offers various analyses for shotgun metagenome data.
+- <!--- What part of the repo is the current PR concerned with? E.g. The current PR addressed a bug in action X, which runs a Y analysis on the inputs. -->
+
+### What's new
+- <!--- Describe what was changed in the code and why it is useful or necessary-->
+- <!--- Does this PR fully address an existing issue? If so write: Closes #issue_number -->
+
+### Additional Info
+<!--- Is the PR blocked by another PR? If so, disclose it here. You can use the syntax user/repo_name/pull/PR_number to reference PRs in other repos. To reference PRs in the same repo simply use #PR_number. Do so inside the ```[tasklist]``` context as shown below.
+```[tasklist]
+### Blocked by
+- [ ] user/repo_name/pull/PR_number
+- [ ] #PR_number
+```
+If you also what the CI tool to block merging of the PR before another you can use the following:
+- blocked by #PR_number
+- merge after user/repo_name#PR_number
+- dependent on #PR_number
+-->
+
+### Set up an environment
+<!---
+- The following commands should get the reviewer a working environment where they can test the PR changes. 
+- Keep in mind that if you PR depends on newer versions of the dependencies you will have to install these manually, e.g: pip install git+https://github.com/username/repository.git or pip install -e . your_local_dependency
+-->
+```bash
+# For linux: 
+# export MY_OS="linux"
+# For mac:
+export MY_OS="osx" 
+wget "https://data.qiime2.org/distro/shotgun/qiime2-shotgun-2023.9-py38-"$MY_OS"-conda.yml"
+conda env create -n q2-shotgun --file qiime2-shotgun-2023.9-py38-osx-conda.yml
+rm "qiime2-shotgun-2023.9-py38-"$MY_OS"-conda.yml"
+```
```

Comment:
Having reviewed a couple of PRs yesterday and today, I am going to argue again that this section should be removed - those instructions should be in the dev section of the README. On top of repetition, the main reason is that there are at least three different OS options (macOS: ARM vs Intel, plus Linux - I don't even mention Windows) so if you provide instructions for _your_ environment, those don't necessarily apply to the reviewer's environment. For example, I'm using an Intel-based Mac but a PR I was recently reviewing had instructions for the ARM-based one - I had to set it up myself anyways.